### PR TITLE
improve apify go actor example

### PR DESCRIPTION
**Pull request by josuebrunel**
show how to access kv_store

### CODE REVIEW

**Comment by jirimoravcik** [comment]
Hello @josuebrunel,
thanks for your contribution!

File: main.go [hunk 1]
```diff
@@ -1,33 +1,104 @@
...
+func scrape(urls []string, exporter export.Exporter) {
+	g := geziyor.NewGeziyor(&geziyor.Options{
+		StartURLs: urls,
+		// StartRequestsFunc: func(g *geziyor.Geziyor) {
+		// 	g.GetRendered(urls[0], quotesParse)
+		// },
+		ParseFunc: quotesParse,
+		Exporters: []export.Exporter{&export.JSON{}, exporter},
+		// BrowserEndpoint: "ws://localhost:3000",
+	})
```
Comment (jirimoravcik) [inline main.go]: Could you please remove the commented out code? thanks

File: main.go [hunk 2]
```diff
@@ -1,33 +1,104 @@
...
-		log.Fatal(err)
+		xlog.Error("failed to get input from kv store", "error", err)
 		return
 	}
+	xlog.Info("input from kv store", "input", input)
+	url := input["url"].(string)
+	if strings.EqualFold(url, "") {
+		xlog.Error("no url in input", "url", url)
+	}
+	// scrape data
```
Comment (jirimoravcik) [inline main.go]:
```suggestion
	// Scrape data
```
Consistency

**Review by jirimoravcik — CHANGES_REQUESTED** [review]

File: main.go (hunk 1 above)
Comment (josuebrunel) [reply main.go]: Yes sure

**Pull request merged**